scripts/check_md5sums.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -e
+
+if [ $# -ne 1 ] || ! { [ "$1" = "zson" ] || [ "$1" = "bzson" ]; }; then
+  echo 'Must specify output type to be cheecked: "zson" or "bzson"'
+  exit 1
+fi
+
+ZNG_TYPE="$1"
+REPO_DIR="$(cd "$(dirname "$0")" && pwd)/.."
+cd "$REPO_DIR"
+TMPFILE=$(mktemp)
+
+for FILE in "$REPO_DIR"/"$ZNG_TYPE"/*
+do
+  COMPARE_TO="$(basename "$FILE")"
+  ZPATH=${COMPARE_TO/.${ZNG_TYPE}.gz/}
+  echo -n "${ZPATH}:" | tee -a "$TMPFILE"
+  gzcat zeek-default/"$ZPATH".log.gz | zq -f "$ZNG_TYPE" - | md5sum | awk '{ print $1 }' | tee -a "$TMPFILE"
```

Comment:
Portability nits: I've found differences in behavior with the convenience *cat and *grep tools on Mac OS vs. GNU/Linux. This works on both, right? And now that I think of it, I don't think Mac OS has `md5sum` by default....

Using `\` would make this more readable also.

Reply:
RE: Portability. Thanks for the reminder. I just looked into this and put in a change to use `gzcat` if available (which it is on Mac OS) and `zcat` if not (since that handles `.gz` on Linux, whereas the `zcat` that ships with Mac OS is `.Z` only). Likewise, you were right about `md5sum` so I rigged it up to use `md5 -q` if there's no `md5sum` in the `PATH`. It now tests out fine on both Mac OS and Linux.

And I've put in the `\` you suggested.